compiler/optimizer/LoopReplicator.cpp

```diff
@@ -314,14 +314,16 @@ bool TR_LoopReplicator::isWellFormedLoop(TR_RegionStructure *region, TR_Structur
 
    if (_maxNestingDepth > MAX_REPLICATION_NESTING_DEPTH)
       {
-      traceMsg(comp(), "for loop (%d), max nest depth thresholds exceeded\n", region->getNumber());
+      if (trace())
+         traceMsg(comp(), "for loop (%d), max nest depth thresholds exceeded\n", region->getNumber());
       return false;
       }
 
    //growth factor, 33% of the loop
    //
    if ((numBlocks*MAX_REPLICATION_GROWTH_FACTOR) > MAX_REPLICATION_GROWTH_SIZE)
       {
+      if (trace())
       traceMsg(comp(), "for loop (%d), loop too big, thresholds exceeded\n", region->getNumber());
```

Comment:
This line needs to be indented.
```suggestion
         traceMsg(comp(), "for loop (%d), loop too big, thresholds exceeded\n", region->getNumber());
```